### ISSUE TITLE
Extra double quote on variation dropdown select

### ISSUE THREAD

**Issue by clescuyer** [woocommerce/woocommerce#12251]
## EXPLANATION OF THE ISSUE

There is an extra " in the <select> tag for wc_dropdown_variation_attribute_options()

File wc-template-functions.php line 2076

-		$html = '<select id="' . esc_attr( $id ) . '" class="' . esc_attr( $class ) . '" name="' . esc_attr( $name ) . '" data-attribute_name="attribute_' . esc_attr( sanitize_title( $attribute ) ) . '"' . '" data-show_option_none="' . ( $show_option_none ? 'yes' : 'no' ) . '">';
+		$html = '<select id="' . esc_attr( $id ) . '" class="' . esc_attr( $class ) . '" name="' . esc_attr( $name ) . '" data-attribute_name="attribute_' . esc_attr( sanitize_title( $attribute ) ) . '" data-show_option_none="' . ( $show_option_none ? 'yes' : 'no' ) . '">';

## STEPS TO REPRODUCE THE ISSUE

Look at HTML generated source, attribute *data-attribute_name*

## SYSTEM STATUS REPORT

```
Environnement WordPress
URL accueil : 		http://shop.terres-lointaines.local
URL du site : 		http://shop.terres-lointaines.local
Version WC : 		2.6.7
Dossier log enregistrable : 		/Users/xtian/development/shop.terres-lointaines.com/wp-content/uploads/wc-logs/
Version WP : 		4.6.1
WP multisite : 		–
Limite mémoire WP : 		256 MB
Mode débogage WP : 		
Cron WP : 		–
Langue : 		fr_FR
```

**Comment by FullPhatDesign** [comment]
Is this fix pushed to the latest version of woocommerce yet? I am on 2.6.14 and am still having the issue, the validator is blowing up on me.

https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.feedyourgoal.com%2Fproduct%2Fanabolic-designs-aminotaur-green-apple%2F

**Comment by ptrkcsk** [comment]
@FullPhatDesign No, but it's fixed in [2.7.0-RC1](https://github.com/woocommerce/woocommerce/blob/2.7.0-RC1/includes/wc-template-functions.php#L2184), so it should arrive in a stable version soon.
